client/blindbox/requests/secure_session.py

```diff
@@ -126,6 +135,78 @@ def request(self, method, endpoint, *args, **kwargs):
         joined_url = urljoin(self.base_url, endpoint)
         return super().request(method, joined_url, *args, **kwargs)
 
+    def attestation(self, policy):
+        import base64
+        import hashlib
+        import json
+        import jwt
+        import secrets
+        from .errors import AttestationException
+
+        decoded_policy = base64.b64decode(policy)
+        h = hashlib.new('sha256')
+        h.update(decoded_policy)
+        cce_policy = h.hexdigest()
+
+        nonce = {"nonce":secrets.token_hex(16)}
+        b64nonce = base64.b64encode(json.dumps(nonce).encode()).decode()
+
+        # Make a request to the attestation service for an MAA token
+        attest_url = ":".join(self.base_url.split(":")[:-1])
+        print(attest_url)
+        res = super().request("POST",f"{attest_url}:8080/attest/maa", json={"maa_endpoint":"blindboxattester.eus.attest.azure.net","runtime_data":b64nonce})
```

Comment:
So you are hardcoding the attestation verification URL? Won't it cause an issue for other regions?

Reply:
Made the attestation endpoint an argument passed for session creation